app/api/v2/views.py

```diff
@@ -0,0 +1,383 @@
+from flask import Flask, jsonify, request, abort, make_response, render_template
+import json
+import datetime
+import jwt
+from functools import wraps
+from .utils.user_utils import UserValidation
+from .utils.party_utils import PartyValidation
+from .utils.office_utils import OfficeValidation
+from .models import Dbase, User, Party, Office, Candidate, Vote
+
+
+def bad_request(message):
+    response = jsonify({
+        "message": message,
+        "status": 400,
+    })
+    response.status_code = 400
+    return response
+
+
+def create_app():
+    app = Flask(__name__)
+    app.config['SECRET_KEY'] = "thisisthesecretkey"
```

Comment:
This needs to move to your env. Consider moving any sensitive data there. DO NOT hard code important, sensitive information